gui/citylist_frame_t.cc

```diff
@@ -102,12 +105,20 @@ citylist_frame_t::citylist_frame_t() :
 
 	list.set_table_layout(1,0);
 
-	list.add_table(3, 3);
+	list.add_table(3, 4);
 	list.new_component<gui_label_t>("Filter:");
 	name_filter_input.set_text(name_filter, lengthof(name_filter));
 	list.add_component(&name_filter_input);
 	list.new_component<gui_fill_t>();
 
+	// add copy csv button
+	bt_copy_csv.init(button_t::roundbox | button_t::flexible, translator::translate("Copy csv format"));
+	bt_copy_csv.set_tooltip("Copy citylist data to clipboard in csv format.");
```

Comment:
Please make it more concrete.
```suggestion
	bt_copy_csv.init(button_t::roundbox | button_t::flexible, translator::translate("Copy city list"));
	bt_copy_csv.set_tooltip("Copy the list of city name and population to clipboard in csv format.");
```